### PR TITLE
Feature/app info

### PR DESCRIPTION
**Pull request by korymarks**
See what you think about using [Heroku Labs: Dyno Metadata](https://devcenter.heroku.com/articles/dyno-metadata) for adding our build number and commit hash to the app info route? 
<hr />
heroku-config-vars

### CODE REVIEW

File: src/services/getAppInfo.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+  const gitInfo = await gitData();
   return new Promise((resolve) => {
     resolve({
       title: pkgJSON.name,
       environment: env,
       version: pkgJSON.version,
-      commit: 'TODO!',
-      build: 'TODO!',
+      commit: config.herokuSlugCommit || gitInfo.long,
+      build: config.herokuSlugDescription || gitInfo.short,
```
Comment (ronwarner1231) [inline src/services/getAppInfo.js]: Seems this should be either HEROKU_RELEASE_VERSION or some other version increment for the API...

Comment (JeremiahDotNet) [reply src/services/getAppInfo.js]: @ronwarner1231 It looks like the version is being pulled from package.json

**Comment by JeremiahDotNet** [comment]
This PR addresses Issue #1

File: src/config.js [hunk 2]
```diff
@@ -6,33 +6,33 @@ const config = cfg({
...
   appName: {
     env: 'APP_NAME',
     type: 'string',
-    default: 'fusion-node-express-api-skeleton',
+    required: true,
   },
   desiredPort: {
     env: 'PORT',
     type: 'integer',
     default: '3000',
```
Comment (BWortman) [inline src/config.js]: We'll need to be careful when merging this file if PR #12 lands first.

File: src/config.js [hunk 3]
```diff
@@ -6,33 +6,33 @@ const config = cfg({
...
-  nodeEnv: {
-    env: 'NODE_ENV',
-    type: 'enum',
-    values: ['development', 'production'],
-    default: 'development',
+  herokuSlugCommit: {
+    env: 'HEROKU_SLUG_COMMIT',
+    type: 'string',
+  },
+  herokuSlugDescription: {
```
Comment (BWortman) [inline src/config.js]: I think we can drop this.

File: src/services/getAppInfo.js (hunk 1 above)
Comment (BWortman) [inline src/services/getAppInfo.js]: We may as well drop the build. It doesn't really add value. The build should be in the version number... which should be incremented by an automated build process (can you say, "out of scope"?).

**Comment by BWortman** [comment]
Great work, @korymarks! I think you can back out the 'build' property and this will be done.

**Pull request merged**